### PR TITLE
Add helper descriptions on new repo page

### PR DESCRIPTION
**Pull request by bagasme**
Add helpers for:
  * repo description
  * .gitignore
  * license
  * README
  * default branch
  * signature trust model

Signed-off-by: Bagas Sanjaya <bagasdotme@gmail.com>

### CODE REVIEW

**Comment by a1012112796** [comment]
@bagasme would you please add some screenshot on pull description. Thanks.

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -684,17 +685,27 @@ use_template = Use this template
...
+license_helper_desc = A license governs what others can and can't do with your code. Not sure which one is right for your project? See <a target="_blank" rel="noopener noreferrer" href="%s">Choose a license.</a>
 readme = README
 readme_helper = Select a README file template.
+readme_helper_desc = This is the place where you can write a complete description for your project.
 auto_init = Initialize Repository (Adds .gitignore, License and README)
+trust_model_helper_intro = Select trust model for signature verification. Possible options are:
+trust_model_helper_collaborator = Collaborator: Trust signatures by collaborators
+trust_model_helper_committer = Committer: Trust signatures that match committers
+trust_model_helper_collaborator_committer = Collaborator+Committer: Trust signatures by collaborators which match the committer
+trust_model_helper_default = Default: Use the default trust model for this installation
```
Comment (a1012112796) [inline options/locale/locale_en-US.ini:708]:
```suggestion
trust_model_helper_default = Default Trust Model: Use the default trust model for this installation
```
the value in choice item.

**Review by a1012112796 — APPROVED** [review]

**Comment by bagasme** [comment]
review ping

**Review by noerw — APPROVED** [review]
looks great! please merge current master

**Pull request merged**